gradle.properties

```diff
@@ -18,3 +18,5 @@ org.gradle.java.installations.auto-detect = true
 # JDK early access versions and Gradle just reports errors when trying to download them.
 org.gradle.java.installations.auto-download = false
 
+# stop nagging about the configuration cache
+org.gradle.configuration-cache=false
```

Comment:
Nitpick: both style of assignment used in this file.

` = ` (spaces around) and `=` (no spaces).

Reply:
Fixed!